src/gocept/gtimelog/core.py

```diff
@@ -155,11 +155,11 @@ def grouped_entries(self, skip_first=True):
                     start = min(start, old_start)
                     duration += old_duration
                 entries[entry] = (start, entry, duration)
-        work = work.values()
+        work = list(work.values())
         work.sort()
```

Comment:
```suggestion
        work = sorted(work.values())
```

How about that? (Same for the following lines.)

Reply:
Done.